src/redux/reservation/thunk.js

```diff
@@ -0,0 +1,37 @@
+// reservationThunks.js
```

Comment:
- [ ] Kindly, remove this piece of old commented code.  that way, the repo's cleanliness will be improved. 👍